core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1967,27 +1967,31 @@ class WorkspaceService(protected val userInfo: UserInfo,
     * @param billingProject
     * @param billingAccount
     * @param workspaceId
+    * @param rbsHandoutRequestId
     * @param span
     * @return Future[(GoogleProjectId, GoogleProjectNumber)] of the project that we claimed from RBS
     */
   def setupGoogleProject(billingProject: RawlsBillingProject,
                          billingAccount: RawlsBillingAccountName,
                          workspaceId: String,
                          workspaceName: WorkspaceName,
-                         span: Span = null) : Future[(GoogleProjectId, GoogleProjectNumber)] = {
-    val projectPoolType = billingProject.servicePerimeter match {
-      case Some(_) => ProjectPoolType.ExfiltrationControlled
-      case _ => ProjectPoolType.Regular
-    }
-
+                         rbsHandoutRequestId: String,
+                         span: Span = null) : Future[(GoogleProjectId, GoogleProjectNumber)] =
     for {
       googleProjectId <- traceWithParent("getGoogleProjectFromBuffer", span) { _ =>
-        resourceBufferService.getGoogleProjectFromBuffer(projectPoolType, workspaceId)
+        resourceBufferService.getGoogleProjectFromBuffer(
+          if (billingProject.servicePerimeter.isDefined)
+            ProjectPoolType.ExfiltrationControlled else
+            ProjectPoolType.Regular,
```

Comment:
nit: in-lining this makes it a bit harder to read

Reply:
I actually find it easier to read as it reduces the number variables in scope that I have to keep track of while reading the code.